atomist/configure/advisories.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Advisory sources
+description:
+keywords:
+---
+
+{% include atomist/disclaimer.md %}
+
+Vulnerabilities and advisories in the Atomist vulnerability database are updated
+continuously. Data from advisories is ingested, processed, enriched, and
+transformed from the following sources, by default:
+
+|           | Name                               | URL                                                   |
+| :-------: | ---------------------------------- | ----------------------------------------------------- |
+| `alpine`  | Alpine `secdb`                     | <https://secdb.alpinelinux.org/?>                     |
+| `amazon`  | Amazon Linux Security Center       | <https://alas.aws.amazon.com/>                        |
+| `debian`  | Debian Security Bug Tracker        | <https://security-tracker.debian.org/tracker/>        |
+|  `nist`   | National Vulnerability Database    | <https://nvd.nist.gov/>                               |
+| `redhat`  | RedHat Security Data               | <https://www.redhat.com/security/data/metrics/>       |
+| `ubuntu`  | Ubuntu CVE Tracker                 | <https://people.canonical.com/~ubuntu-security/cve/>  |
+|  `suse`   | SUSE Security CVRF                 | <http://ftp.suse.com/pub/projects/security/cvrf/>     |
+| `github`  | GitHub Advisory Database           | <https://github.com/advisories/>                      |
+| `gitlab`  | GitLab Advisory Database           | <https://gitlab.com/gitlab-org/advisories-community/> |
+| `golang`  | Golang VulnDB                      | <https://github.com/golang/vulnbd>                    |
+| `rustsec` | RustSec Advisory Database          | <https://github.com/rustsec/advisory-db>              |
+|  `pypa`   | Python Packaging Advisory Database | <https://github.com/pypa/advisory-database>           |
+
+The ingestion process for vulnerabilities and advisories operates around the
```

Comment:
From here on, the page content should be removed as it is very much intended for internal audiences.